src/documents/tutorial-prefsFramework/CreatingAPrefsEditor.md

```diff
@@ -0,0 +1,779 @@
+Using this grade automatically buys you a lot of Framework supports necessary for Panels.
+</dd>
+<dt>`preferenceMap`</dt>
+<dd>A Panel must have a _ Preference Map_, which maps the information in the Primary Schema
+into your Panel.Let’s look at this one more closely:
+<pre class="highlight">
+<code class="hljs javascript">{
+preferenceMap: {
+    "minEditor.autoPilot": {
+        "model.autoPilot": "default"
+    }
+},</code></pre>
+
+The first line of the  Preference Map, `“minEditor.autoPilot”`, is the name of the preference.
+This exactly matches the name we saw in the Primary Schema earlier. The value for this
+key is a JavaScript object that defines how this particular preference relates to the Panel’s
+internal data model.
+
+The content of this  Preference Map is a key/value pair:
+<ul>
+<li> The key, `“model.autoPilot”`, is an [EL path](../FrameworkConcepts.md#el-paths) into the
+Panel’s data model. An “EL path” is just a
+dot-separated path built from names. In this case, it means “the `autoPilot`
+property of the [`model`](../FrameworkConcepts.md#model-objects) property” of the Panel.
+<aside class="infusion-docs-callout">
+[Models](../FrameworkConcepts.md#model-objects) are central to Infusion, which,
+while not formally a [Model-View-Controller framework](../FrameworkConcepts.md#mvc),
+embodies the the separation of concerns that is central to MVC.
+Most Infusion components have an internal model, for maintaining the state of the component.
+</aside>
+</li>
+<li> The value, `“default”`, is an EL path referencing the `“default”` property in the Primary Schema.</li>
+</ul>
+This  Preference Map is saying two things:
+<ol>
+<li>The preference called `“minEditor.autoPilot”` should be stored in the Panel’s model
+in a property called `autoPilot`, and</li>
+<li>the initial value for the property should be taken from the `“default”` property
+of the Primary Schema.</li>
+</ul>
+A Preference Map can specify other locations for Primary Schema information, besides the model.
+We'll see an example of this when we add antoher panel, later in this tutorial.
+</dd>
+<dt>`selectors`</dt>
+<dd>A Panel is a _[view component](../tutorial-gettingStartedWithInfusion/ViewComponents.md)_
+– a type of Infusion component that has a _view_, that is,
+a user interface. In order to maintain a separation between the code and the HTML for the view,
+the code interacts with the HTML through named [selectors](../ComponentConfigurationOptions.md#-selectors-): The code only references the name, and
+a Framework feature called the DOM Binder looks up the relevant DOM node for the name based
+on the information in this `selectors` option.
+
+Let’s look at this more closely:
+<pre class="highlight">
+<code class="hljs javascript">selectors: {
+    autoPilot: ".mec-autoPilot"
+},</code></pre>
+
+The content of a `selectors` property is a set of key/value pairs. The key is the ‘name’ of the
```

Comment:
Are you using single quotes vs double quotes in a specific way here. (i.e 'name'). If so, should that go in the style guide too?